requirements.txt

```diff
@@ -2,4 +2,8 @@ beautifulsoup4
 requests
 retrying
 pysocks
-pandas
\ No newline at end of file
+pandas
```

Comment:
Do you need pandas? Or you would get it from Python Slim version from Alpine or other distribution when making a DockerFile later.

Reply:
I am using pandas to read from csv and using map operation for a function. I will replace with native python, if i am not going to use it for other purposes.
If I am going to use it, I will try Alpine.